codelabs/custom_generator/custom_generator.md

```diff
@@ -286,12 +286,13 @@ sampleGenerator['left_turn_block'] = function(block) {
 Value blocks represent code that returns a value.
 
 A value block's generator returns an array containing a string and a [precedence value](https://developers.google.com/blockly/guides/create-custom-blocks/operator-precedence).
+The built-in generators have predefined operator precedence values exported as an `Order` enum.
```

Comment:
Changed, but in general not interested in hand-formatting files unless it's really egregious :P

Reply:
The question was more about whether you had omitted the paragraph break, rather than requesting a formatting change—but I can see that I could have worded it more clearly.